share/nisar/defaults/gcov.yaml

```diff
@@ -570,14 +570,27 @@ runconfig:
                 # Default: True
                 qa_reports: true
 
+            software_config:
+
+                # True to use memory map(s) to cache select Dataset(s).
+                # False to always read data directly from the input file.
```

Comment:
What does "directly" mean here? I thought memory mapping was applied directly from the files. Or does this process 'flatten' the data first to make it memory-mappable?